Remove nonexistent ws.port value from Erigon allowedTCPPorts

RichardWarfield opened the pull request:
Currently the module for Erigon adds the value of the `ws.port` option to `allowedTCPPorts`, but this option does not exist either as an option in the module, nor on the Erigon command line.  Because the option does not exist, starting Erigon with `openFirewall = true` currently fails.  This PR removes the reference to `ws.port` in `allowedTCPPorts`.

aldoborrero commented:
Hi, @RichardWarfield thanks for your contributions! In order to keep the flow going I think we can `cherry-pick` your commits and prepare separate PRs so we get those commits in the main branch so everybody can benefit us. 

@selfuryon can you help me with this?

RichardWarfield commented:
Please feel free.  Apologies, I've been travelling lately and just haven't
been able to attend to this. I actually didn't realize that pushing to the
branch would also enlarge the PR (oops).  I've added modules for some
additional components (prysm validator, mev boost) but please check whether
these are implemented the way you would like.  For example, I've added an
option to set the user and datadir for the Prysm services so the validator
and beacon clients can be set to the same user and access the same data
dir.  There may be a cleaner way to do this while continuing to use the
systemd state dir as the datadir but I'm not such an expert on systemd
options.

On Fri, Apr 28, 2023 at 3:41 PM Aldo ***@***.***> wrote:

> Hi, @RichardWarfield <https://github.com/RichardWarfield> thanks for your
> contributions! In order to keep the flow going I think we can cherry-pick
> your commits and prepare separate PRs so we get those commits in the main
> branch so everybody can benefit us.
>
> @selfuryon <https://github.com/selfuryon> can you help me with this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nix-community/ethereum.nix/pull/230#issuecomment-1527198979>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7JIMEBUBDLSGZTNMU2L73XDN7EVANCNFSM6AAAAAAXH4PR2A>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

aldoborrero commented:
Closing this PR in favor of #280 and #279 (let's move the discussion there, if necessary)
